### PR TITLE
Update dependencies - Closes #746

### LINKED ISSUE #746: Update dependencies
### Expected behavior

Our dependencies should be updated for new releases.

### Which version(s) does this affect? (Environment, OS, etc...)

1.1.0

### PR DESCRIPTION
**Pull request by willclarktech**
### What was the problem?

Our dependencies hadn't been updated.

### How did I fix it?

Updated them.

### How to test it?

Install, build, lint, test etc.

### Review checklist

* The PR resolves #746 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: packages/lisk-cryptography/.eslintrc.json [hunk 1]
```diff
@@ -2,5 +2,8 @@
 	"extends": ["../../.eslintrc.json"],
 	"globals": {
 		"naclInstance": true
+	},
+	"rules": {
+		"import/no-cycle": "off"
```
Comment (shuse2) [inline packages/lisk-cryptography/.eslintrc.json]: 😭

Comment (willclarktech) [reply packages/lisk-cryptography/.eslintrc.json]: 😯 I'll create a follow-up issue and add it to 1.1.0.

Comment (willclarktech) [reply packages/lisk-cryptography/.eslintrc.json]: https://github.com/LiskHQ/lisk-elements/issues/775

File: Jenkinsfile [hunk 2]
```diff
@@ -32,6 +32,11 @@ pipeline {
 					cache_file = restoreCache("package.json")
 					sh 'npm install --verbose'
 					saveCache(cache_file, './node_modules', 10)
+					sh '''
+					if [ ! -f /home/lisk/.cache/Cypress/3.1.0/Cypress/Cypress ]; then
```
[the remark this answers is not in the document]
Comment (willclarktech) [reply Jenkinsfile]: @fchavant Updated.

File: Jenkinsfile [hunk 3]
```diff
@@ -32,6 +32,11 @@ pipeline {
 					cache_file = restoreCache("package.json")
 					sh 'npm install --verbose'
 					saveCache(cache_file, './node_modules', 10)
+					sh '''
+					if [ ! -f "/home/lisk/.cache/Cypress/$(cat ./packages/lisk-constants/package.json| jq -r '.devDependencies.cypress')/Cypress/Cypress" ]; then
```
Comment (fchavant) [inline Jenkinsfile]: How about just `jq -r .devDependencies.cypress ./packages/lisk-constants/package.json`? No useless use of cat and a little shorter.

Comment (willclarktech) [reply Jenkinsfile]: @fchavant Updated

**Pull request merged**